api/tables/store.js

```diff
@@ -49,22 +49,26 @@ export function createStoreTable (region, tableName, options = {}) {
     /**
      * Bind a link CID to an account
      *
+     * @typedef {import('../service/types').StoreItemOutput} StoreItemOutput
+     * 
      * @param {import('../service/types').StoreItemInput} item
+     * @returns {Promise<StoreItemOutput>}
      */
-    insert: async ({ uploaderDID, link, proof, origin, size = 0 }) => {
+    insert: async ({ space, link, origin, size = 0, issuer, invocation }) => {
+      /** @type StoreItemOutput */
       const item = {
-        uploaderDID,
-        payloadCID: link,
-        applicationDID: '',
-        origin: origin || '',
+        space,
+        link: link.toString(),
         size,
-        proof,
-        uploadedAt: new Date().toISOString(),
+        origin: origin?.toString(),
+        issuer,
+        invocation: invocation.toString(),
+        insertedAt: new Date().toISOString(),
```

Comment:
My mind is wanting single word since everything else is - inserted?

Reply:
I vote we keep it as `insertedAt` to align with the `access-api` db schema


```sql
CREATE TABLE
    IF NOT EXISTS spaces (
        did TEXT NOT NULL PRIMARY KEY,
        product TEXT NOT NULL,
        email TEXT NOT NULL,
        agent TEXT NOT NULL,
        inserted_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
        updated_at TEXT NOT NULL DEFAULT (strftime ('%Y-%m-%dT%H:%M:%fZ', 'now')),
        UNIQUE (did)
    );
```


https://github.com/web3-storage/w3protocol/blob/77b999e2f4c40f1f0b5397db6b1f8ee99232b85e/packages/access-api/migrations/0000_create_spaces_table.sql#LL8